src/services/domain/IDomain.ts

```diff
@@ -1,6 +1,5 @@
-import {IServerAddress} from "../server/IServerAddress";
 
 export interface IDomain {
   controller: string;
-  servers: IServerAddress[]; // Map of format hostname: [server1, server2]
+  servers: string[][]; // Map of format hostname: [server1, server2]
```

Comment:
Typedoc flags it as an incompatible assignment. I tested in a debugger and it indeed is. We are not dealing here with IServerAddress types but with an array of string arrays. Please see it for your self and check the types